airflow/configuration.py

```diff
@@ -324,17 +344,47 @@ def run_command(command):
 # how often the scheduler should run (in seconds).
 scheduler_heartbeat_sec = 5
 
+# after how much time should the scheduler terminate in seconds
+# -1 indicates to run continuously (see also num_runs)
+run_duration = -1
+
+# after how much time a new DAGs should be picked up from the filesystem
+min_file_process_interval = 0
```

Comment:
This is the amount of time we should wait between ("processing")[https://github.com/apache/incubator-airflow/blob/1.8.0/airflow/jobs.py#L1496-L1519] a dag run 

https://github.com/apache/incubator-airflow/blob/1.8.0/airflow/utils/dag_processing.py#L591-L596